met-web/src/components/layout/Header/InternalHeader.tsx

```diff
@@ -54,66 +51,37 @@ const InternalHeader = ({ drawerWidth = 280 }: HeaderProps) => {
                             <FontAwesomeIcon icon={faBars} style={{ fontSize: '20px' }} />
                         </IconButton>
                     </When>
-                    <When condition={logoUrl && !imageError}>
-                        <Box
-                            sx={{
-                                backgroundImage: logoUrl,
-                                height: '5em',
-                                width: { xs: '7em', md: '15em' },
-                                marginRight: { xs: '1em', md: '3em' },
-                            }}
-                        >
-                            <img
-                                src={logoUrl}
-                                alt="Site Logo"
-                                style={{
-                                    objectFit: 'cover',
-                                    height: '5em',
-                                    width: '100%',
-                                    cursor: 'pointer',
-                                }}
-                                onClick={() => {
-                                    navigate('/home');
-                                }}
-                                onError={(_e) => {
-                                    setImageError(true);
-                                }}
-                            />
-                        </Box>
-                    </When>
-                    <When condition={!logoUrl || imageError}>
-                        <Box
-                            component={BCLogo}
-                            sx={{
-                                cursor: 'pointer',
-                                height: '5em',
-                                width: { xs: '7em', md: '15em' },
-                                marginRight: { xs: '1em', md: '3em' },
-                            }}
-                            onClick={() => {
-                                navigate('/home');
-                            }}
-                            alt="British Columbia Logo"
-                        />
-                    </When>
+                    <Box
+                        component={BCLogo}
+                        sx={{
+                            cursor: 'pointer',
+                            height: '5em',
+                            width: { xs: '7em', md: '15em' },
+                            marginRight: { xs: '1em', md: '3em' },
+                        }}
+                        onClick={() => {
+                            navigate('/home');
+                        }}
+                        alt="British Columbia Logo"
+                    />
                     {isMediumScreen ? (
-                        <HeaderTitleOld
+                        <Header1
```

Comment:
If you make any change outside of the scope of the ticket, just be sure to document it in the PR description. So for this you could write something like:

`- continue to gradually replace old components with new MET redesign components`